public/app/features/plugins/admin/pages/Browse.tsx

```diff
@@ -11,10 +11,11 @@ import { HorizontalGroup } from '../components/HorizontalGroup';
 import { usePlugins } from '../hooks/usePlugins';
 import { useHistory } from '../hooks/useHistory';
 import { Plugin } from '../types';
-import { Page } from 'components/Page';
+import { Page as PluginPage } from '../components/Page';
 import { CatalogTab, getCatalogNavModel } from './nav';
+import { Page } from 'app/core/components/Page/Page';
 
-export const Browse = ({ meta, onNavChanged, basename }: AppRootProps) => {
+export default function Browse(): JSX.Element | null {
```

Comment:
Do we need the `| null` throughout these page components? It looks like they are always returning a component.